lib/handlers/get.js

```diff
@@ -176,7 +176,7 @@ function globHandler (req, res, next) {
       var data = $rdf.serialize(
         undefined,
         globGraph,
-        null,
+        filename,
         'text/turtle')
```

Comment:
We can open an issue to add support for multiple content types for globbing, for solid-server. This PR is just intended to upgrade the server to the latest rdflib version.